projects/react/src/lib/useMaskito.ts

```diff
@@ -0,0 +1,56 @@
+import {Maskito, MASKITO_DEFAULT_OPTIONS, MaskitoOptions} from '@maskito/core';
+import {RefCallback, useCallback, useState} from 'react';
+
+import {useIsomorphicLayoutEffect} from './useIsomorphicLayoutEffect';
+
+export type MaskitoPredicate = (
```

Comment:
Looks like we got the same type in both packages, maybe move it to core?

Reply:
it does repeat, but 
1. API can change over time and can differ in packages in the future
2. it will add size to core package and it doesn't seem to be helpful when using only core package

Reply:
Type does not add size and a function is rather small. It would be great to keep as little discrepancy as possible between implementations. @nbarsukov, what do you think?

Reply:
comment about size applies only to default predicate

type can imo can be common, yes

but moving defaultMaskitoPredicate to core doesn't seem needed as i commented above in https://github.com/Tinkoff/maskito/pull/273#discussion_r1180076265

Reply:
Just seems like if one would ever make a vue implementation or any other solution, any indirect use would require it.

Reply:
in React we have access to native DOM element reference, same with Angular, but what if in other framework it will differ? type and function would not apply, because predicate is not framework-agnostic

Reply:
I think that we can move type `MaskitoPredicate` to `@maskito/core`, yep.
It will help to "to keep as little discrepancy as possible between implementations".
___
About `defaultMaskitoPredicate` function. It is very simple function. I think we can keep it as it is (without moving it to any Public API).

Reply:
The main problem that `MaskitoPredicate` is not the best naming.
It is not obvious what it means without any context.

Should we rename it to `MaskitoElementPredicate` ?

Reply:
>https://github.com/Tinkoff/maskito/pull/273#discussion_r1180178268

and if in some other library implementation it will store element not as native dom ref

`MaskitoPredicate` explicitly refers to `HTMLElement` because React hook implemented that way, but we can't know how it will be in other frameworks

Reply:
I think if we face any other framework (which can't query element), we will just not use this type :)

Reply:
It's `HTMLInputElement` because that's what we have to feed to the maskito constructor. Still sounds to me like this could be useful and not much of a burden on the core package, especially considering tree shaking. But that's up to you guys, just my thoughts.

Reply:
done https://github.com/Tinkoff/maskito/pull/273/commits/1de95bf38674a51d5614e3d19e4debfbe761df7f